app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,30 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  const [lowerBoundError, setLowerBoundError] = useState('')
+  const [startingPointError, setStartingPointError] = useState('')
+  const [upperBoundError, setUpperBoundError] = useState('')
+
+  useEffect(() => {
+    Number(lowerBound) < 0
```

Comment:
Should be `lowerBound.lt(Zero)` here with `Zero` being imported from `ethers/constants` (could also use `new BigNumber(0)` as it's the same value, but `Zero` is a bit cleaner)

Reply:
oops, good catch. missed that one on line 128. 

added these changes